### ISSUE TITLE
[Bug Report] Issue with Chrome Lighthouse and v-rating - Buttons do not have an accessible name

### ISSUE THREAD

**Issue by Ogar616** [vuetifyjs/vuetify#11590]
### Environment
**Vuetify Version:** 2.2.33
**Vue Version:** 2.6.11
**Browsers:** Chrome 83.0.4103.61
**OS:** Mac OS 10.15.5

### Steps to reproduce
Just use v-rating component, and test your project with Chrome Lighthouse.

### Expected Behavior
Stars - buttons with v-icon class should have aria-label fe. aria-label="rating-star" even if rating is readonly.

### Actual Behavior
Chrome Lighthouse is showing accessibility issue with single rating stars:
Buttons do not have an accessible name.

Failing Elements:
button.v-icon.notranslate.v-icon--link.v-icon--svg.theme--light

<button type="button" tabindex="-1" class="v-icon notranslate v-icon--link v-icon--svg theme--light" style="color:rgba(242,159,5,1);caret-color:rgba(242,159,5,1);">
  

### Reproduction Link
<a href="https://codepen.io/Ogar616/pen/OJMMgvE" target="_blank">https://codepen.io/Ogar616/pen/OJMMgvE</a>

### Other comments
Possible workaround:
Manually add an area-label to buttons with class v-icon like: 

const buttons = document.getElementsByTagName('button');
Array.from(buttons).map((btn) => btn.setAttribute('aria-label', 'rating-star'));

in mounted hook.

Possible fix:
add "aria-label: 'rating-star' " to attributes in:

vuetify/packages/vuetify/src/components/VRating/VRating.ts 
Line 222.
